table.go

```diff
@@ -1626,8 +1641,10 @@ func (t *Table) memoryBlocks() ([]*TableBlock, uint64) {
 	}
 
 	lastReadBlockTimestamp := t.active.ulid.Time()
+	t.active.pendingReadersWg.Add(1)
 	memoryBlocks := []*TableBlock{t.active}
 	for block := range t.pendingBlocks {
+		block.pendingReadersWg.Add(1)
```

Comment:
Add a comment to `memoryBlocks` indicating that that `Done` must be called on the block's waitgroup.